src/wristpy/core/orchestrator.py

```diff
@@ -0,0 +1,259 @@
+"""Python based runner."""
+
+import pathlib
+from typing import Any, Dict, List, Optional
+
+import numpy as np
+import polars as pl
+from pydantic import BaseModel
+
+from wristpy.core import computations, config, models
+from wristpy.io.readers import readers
+from wristpy.processing import analytics, calibration, metrics
+
+
+class InvalidFileTypeError(Exception):
+    """Wristpy cannot save in the given file type."""
+
+    pass
+
+
+class Results(BaseModel):
```

Comment:
Keep an eye on being consistent with import styling i.e. this would be pydantic.BaseModel if you use the module imports everywhere else.